web/src/components/RunBlockEditor.tsx

```diff
@@ -415,7 +415,7 @@ function RunBlockPlateSamplerEditor({ disabled, definition, outputPlateLabel, se
         const label = plates && plates[i];
         const id = plateIds && plateIds[i];
         inputRows.push(<tr key={i}>
-            <th>Input Plate {i}</th>
+            <th>Input Plate {i+1} <img src={"../images/quadrant_" + i + ".png"} width="75"/> </th>
```

Comment:
Ideally this image should be colored divs so its easy to resize it/make it responsive, but for now, lets go with this.